engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -117,7 +117,7 @@ object WorkflowActor {
       // This only does the initialization for a newly created workflow.  For a restarted workflow we should be able
       // to assume the adjusted symbols already exist in the DB, but is it safe to assume the staged files are in place?
       actor.initializeWorkflow match {
-        case Success(inputs) => actor.createWorkflow(inputs)
+        case Success(_) => actor.createWorkflow(actor.workflow.actualInputs)
```

Comment:
why do we need to call a method on the actor parameterized by data on the actor?